pkg/apis/controlplane/v1beta2/generated.proto

```diff
@@ -78,9 +78,13 @@ message AppliedToGroupPatch {
 message ClusterGroupMembers {
   optional k8s.io.apimachinery.pkg.apis.meta.v1.ObjectMeta metadata = 1;
 
-  repeated GroupMember effectiveMembers = 2;
+  optional int64 totalMembers = 2;
```

Comment:
Consider changing `totalMembers` and `currentPage` fields to 4 and 5 in protobuf message for backward comp